R/o2.at.sat.R

```diff
@@ -60,46 +86,73 @@ o2.at.sat <- function(ts.data, baro, altitude=0, salinity=0, model='garcia'){
 	return(data.frame(datetime=ts.data$datetime, do.sat=dosat))  
 }
 
-#'@export
+
+#' @rdname o2.at.sat
+#' @export
 o2.at.sat.base <- function(temp, baro, altitude=0, salinity=rep(0,length(temp)), model='garcia'){
```

Comment:
I do think that garcia-benson is the preferred model. I kept the default at garcia to keep things from shifting (even more) under current user's feet, but if you're good with garcia-benson as the default, I'll change it.